### PR TITLE
[bitnami/sealed-secrets] Fix combination of additionalNamespaces, rbac.clusterRole=false and rbac.namespacedRoles=true

### PR DESCRIPTION
**Pull request by BWagenerGenerali**
### Description of the change

Currently the sealed-secret chart fails to render with the following minimal values:
```yaml
additionalNamespaces: ["test"]
rbac:
  clusterRole: false
  namespacedRoles: true
```

This configuration causes namespaced roles to be created for every entry in `additionalNamespaces`. Within the definition of the namespaced roles some `include`s are called using `.` instead of `$`. Since this is happening within a `range` `.` does not reference the root context anymore.

The error message produced:

`Error: template: sealed-secrets/templates/rolebinding.yaml:55:24: executing "sealed-secrets/templates/rolebinding.yaml" at <include "sealed-secrets.namespacedRoleName" .>: error calling include: template: sealed-secrets/templates/_helpers.tpl:46:14: executing "sealed-secrets.namespacedRoleName" at <.Values.rbac.namespacedRolesName>: can't evaluate field Values in type string`

This PR fixes the creation of namespaced roles and role bindings.

### Benefits

The creation of namespaced roles and role bindings for `additionalNamespaces` works.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/sealed-secrets/templates/rolebinding.yaml [hunk 1]
```diff
@@ -64,11 +64,11 @@ metadata:
...
   apiGroup: rbac.authorization.k8s.io
   kind: Role
-  name: {{ printf "%s" (include "sealed-secrets.namespacedRoleName" .) }}
+  name: {{ printf "%s" (include "sealed-secrets.namespacedRoleName" $) }}
 subjects:
   - apiGroup: ""
     kind: ServiceAccount
     name: {{ include "sealed-secrets.serviceAccountName" $ }}
-    namespace: {{ include "sealed-secrets.namespace" $ }}
+    namespace: {{ include "common.names.namespace" $ | quote }}
```
Comment (juan131) [inline bitnami/sealed-secrets/templates/rolebinding.yaml:72]: Then we can remove `sealed-secrets.namespace` function from the `_helpers.tpl`, right?

**Review by juan131 — COMMENTED** [review]
Thanks so much for your contribution! Please check my comments.

File: bitnami/sealed-secrets/templates/rolebinding.yaml (hunk 1 above)
Comment (BWagenerGenerali) [reply bitnami/sealed-secrets/templates/rolebinding.yaml:72]: I can still find 23 matches for the include "sealed-secrets.namespace" in other files.
Maybe it's better to replace those usages including this one in a separate PR.

What do you think?

**Review by juan131 — APPROVED** [review]
LGTM

**Comment by BWagenerGenerali** [comment]
Great! There's one failing check: `[CI/CD] CI Pipeline / VIB Verify (pull_request_target) Failing after 55m`

Is there anything I can do about that?

**Comment by juan131** [comment]
> Great! There's one failing check: `[CI/CD] CI Pipeline / VIB Verify (pull_request_target) Failing after 55m`
> 
> Is there anything I can do about that?

We were solving internal issues in the pipeline, sorry for the inconveniences.

**Pull request merged**